Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM alpine:latest
+
+LABEL maintainer="pacocp5@correo.ugr.es" \
+      version="v0.0.6"
+
+RUN apk add --update \
+                    ruby \
+                    make; \
```

Comment:
No pongáis los saltos de línea arbitrariamente. La buena práctica es a partir de 80 caracteres.